src/test/java/org/mifos/integrationtest/cucumber/stepdef/VoucherManagementStepDef.java

```diff
@@ -310,6 +310,21 @@ public void createAndActivateVoucher(){
         iCanCreateAnVoucherRequestDTOForVoucherActivation();
         iCallTheActivateVoucherAPIWithExpectedStatusOfAndStub(202, "/activateVoucher");
     }
+    @Before("@createVoucher")
```

Comment:
instead of before, cant you add this as a step in the scenario itself?

Reply:
the length of feature file is not an issue. the feature file should be more verbose so that everyone can read the test case